internal/testutil/oidctestutil/oidctestutil.go

```diff
@@ -1085,6 +1104,12 @@ func validateAuthcodeStorage(
 		actualDownstreamIDTokenGroups := actualClaims.Extra["groups"]
 		require.Nil(t, actualDownstreamIDTokenGroups)
 	}
+	if len(wantAdditionalClaims) > 0 {
+		actualAdditionalClaims, ok := actualClaims.Get("additionalClaims").(map[string]interface{})
+		require.True(t, ok, "expected additionalClaims to be a map[string]interface{}")
+		require.Equal(t, wantAdditionalClaims, actualAdditionalClaims)
+	}
```

Comment:
Should this make any assertions in an "else" case? Like that the additionalClaims claim is not present?